libcnb-data/src/launch.rs

```diff
@@ -60,29 +60,31 @@ pub struct Label {
     pub value: String,
 }
 
-#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
 #[serde(deny_unknown_fields)]
 pub struct Process {
     pub r#type: ProcessType,
     pub command: String,
-    pub args: Vec<String>,
-    pub direct: bool,
-    #[serde(default)]
-    pub default: bool,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
What does serde output if the `skip_serializing_if = "Option::is_none"` isn't used?

At a guess, I would have thought `None` would serialize to nothing by default, and thus the `skip_serializing_if` here might be redundant?

Reply:
FWIW, the docs mention this case explicitly: https://serde.rs/field-attrs.html#skip_serializing_if. I haven't tested this, but I assume `None` will become `null`.